hooks/lib/ecr-registry-provider.bash

```diff
@@ -1,5 +1,16 @@
 login() {
-  $(aws ecr get-login --no-include-email)
+  local account_id=$(aws sts get-caller-identity --query Account --output text)
+  local region=$(get_ecr_region)
```

Comment:
ShellCheck is anal about these lines (how good would CI be):

```console
Declare and assign separately to avoid masking return values. shellcheck(SC2155)
```

```suggestion
  local account_id
  local region
  
  account_id=$(aws sts get-caller-identity --query Account --output text)
  region=$(get_ecr_region)
```